packages/components/addon/components/hds/flyout/index.js

```diff
@@ -98,6 +105,9 @@ export default class HdsFlyoutIndexComponent extends Component {
     this.element.showModal();
     this.isOpen = true;
```

Comment:
(non-blocking comment) fwiw, I usually see the things that are tracked first inside the action, so I'd probably switch lines 98 and 99. I don't know if this is just a personal preference, or how I learned to do it, or if there's some reason for it but I thought that I'd mention it for your consideration.